base/irrationals.jl

```diff
@@ -19,7 +19,7 @@ for integers `n` will give a rational result when `n` is a perfect square), then
 abstract type AbstractIrrational <: Real end
 
 """
-    Irrational{sym} <: AbstractIrrational
+`Irrational{sym}` <: [`AbstractIrrational`](@ref)
```

Comment:
oh, I didn't necessarily mean we should do this now. I'm probably in favor, but we should let others respond first in a separate PR

Reply:
Sorry, now reverted. 

Possibly it could be done at a higher level, too, just make all types in docstrings clickable somehow?